backend/fms_core/serializers.py

```diff
@@ -783,9 +785,14 @@ def get_steps_order(self, instance):
         return serialized_data.data
 
 class ReferenceGenomeSerializer(serializers.ModelSerializer):
+    editable = serializers.BooleanField(read_only=True)
+    taxon_id = serializers.IntegerField(read_only=True, source='taxon.id')
     class Meta:
         model = ReferenceGenome
-        fields = ("id", "assembly_name", "synonym", "genbank_id", "refseq_id", "taxon_id", "size")
+        fields = "__all__"
+        extra_fields = ("taxon_id", "editable")
```

Comment:
Taxon id is present twice once as taxon and once as taxon_id editable is not present ... This would suggest the extra_field are returned but editable is a BooleanField() not a SerializerMethodField() (the boolean field is tied to no value).